examples/python/example_vamana_dynamic.py

```diff
@@ -118,7 +121,9 @@ def run():
     # Compare with the groundtruth.
     recall = svs.k_recall_at(groundtruth, I, 10, 10)
     print(f"Recall = {recall}")
-    assert(recall == 0.8202)
+    expected_recall = 0.8202
+    assert recall < expected_recall + epsilon
+    assert recall > expected_recall - epsilon
```

Comment:
If we are keeping the `assert_equal` function, this looks like a good place to use it.
```suggestion
    assert_equal(recall, 0.8202)
```

Reply:
That's a great idea, fixed!